docs/sources/introduction/oss-details.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "What is Grafana OSS"
+weight = 5
+aliases = ["/docs/grafana/latest/guides/what-is-grafana"]
++++
+
+# What is Grafana OSS
+
+[Grafana open source software](https://grafana.com/oss/) allows you to query, visualize, alert on, and understand your data no matter where it’s stored. With Grafana you can create, explore and share all of your data through beautiful, flexible dashboards.
```

Comment:
This is great. Can we use it in the _index file?

Reply:
I was trying to avoid duplicate content. Let's keep this as is, can always update the docs later.